tools/harness-thci/OpenThread.py

```diff
@@ -1416,6 +1415,10 @@ def resetAndRejoin(self, timeout):
         try:
             self.powerDown()
             time.sleep(timeout)
+
+            if self.deviceRole == Thread_Device_Role.SED:
+                self.__setPollPeriod(self.__sedPollPeriod)
```

Comment:
What about moving the polling period setting to `powerUp`?

Reply:
I think it can be moved. Should I BTW also remove duplicated call of `__startOpenThread()` in `resetAndRejoin`?

Reply:
Yes please.

Reply:
Done